Subject: [PATCH] Add support for autoQuit
---
Index: src/main/java/org/springframework/samples/petclinic/PetClinicApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/springframework/samples/petclinic/PetClinicApplication.java b/src/main/java/org/springframework/samples/petclinic/PetClinicApplication.java
--- a/src/main/java/org/springframework/samples/petclinic/PetClinicApplication.java	(revision 253a64284a601d5796fe978d4677dce7c14fc274)
+++ b/src/main/java/org/springframework/samples/petclinic/PetClinicApplication.java	(date 1695813974952)
@@ -25,7 +25,14 @@
 public class PetClinicApplication {
 
 	public static void main(String[] args) {
+		long begin = System.currentTimeMillis();
 		SpringApplication.run(PetClinicApplication.class, args);
+		long end = System.currentTimeMillis();
+		boolean quit = Boolean.getBoolean("autoQuit");
+		if (quit) {
+			System.out.println("#### Booted and returned in " + (end - begin) + "ms");
+			System.exit(0);
+		}
 	}
 
 }
